acct_mgt/app.py

```diff
@@ -142,6 +142,8 @@ def get_moc_project(project_uuid):
     def create_moc_project(project_uuid, user_name=None):
         # first check the project_name is a valid openshift project name
         suggested_project_name = shift.cnvt_project_name(project_uuid)
+        annotations = request.get_json(silent=True)
```

Comment:
Remove this, you're not using it later on.

Reply:
fixed